scripts/gh_matrix_builder.py

```diff
@@ -114,6 +124,10 @@ def macos_config(overrides):
 
 m["include"].append(build_release_config(macos_config({})))
 
+m["include"].append(build_without_telemetry({"pg":PG12_LATEST}))
```

Comment:
Do we really want to inflate PR build and test time to check building without telemetry. This serves no useful purpose. Is the plan to keep this when merging the PR?

I suggest moving this to release tests before merge, if that's not already the plan.